opentelemetry-dotnet-contrib.sln

```diff
@@ -261,7 +261,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Shared", "Shared", "{1FCC8E
 		src\Shared\ServerCertificateValidationProvider.cs = src\Shared\ServerCertificateValidationProvider.cs
 		src\Shared\SpanAttributeConstants.cs = src\Shared\SpanAttributeConstants.cs
 		src\Shared\SpanHelper.cs = src\Shared\SpanHelper.cs
-        src\Shared\SqlProcessor.cs = src\Shared\SqlProcessor.cs
+		src\Shared\SqlProcessor.cs = src\Shared\SqlProcessor.cs
```

Comment:
@alanwest FYI seems this whitespace was causing VS to throw errors when opened 🤷